handlers/stations.go

```diff
@@ -319,6 +319,10 @@ func (sh StationsHandler) CreateStation(c *gin.Context) {
 }
 
 func (sh StationsHandler) RemoveStation(c *gin.Context) {
+	if configuration.SANDBOX_ENV == "true" {
```

Comment:
this code is copy pasted a lot of time please move it to a function inside the sandbox handler file, call it denyForSandboxEnv()